pubsub/gossipsub/README.md

```diff
@@ -237,12 +243,12 @@ router, as part of the pubsub api:
 Note that the router can publish messages without topic membership. In order
 to maintain stable routes in that case, it maintains a list of peers for each
 topic it has published in the `fanout` map. If the router does not publish any
-messages for some time, then the `fanout` peers for the topic are forgotten, so
-this is soft state.
+messages of a topic for some time, then the `fanout` peers for that topic are 
+forgotten, so this is soft state.
 
 Also note that as part of the pubsub api, the peer emits `SUBSCRIBE`
 and `UNSUBSCRIBE` control messages to all its peers whenever it joins
-or leaves a topic. This is provided by the the ambient peer discovery
+or leaves a topic. This is provided by the the ambient [peer discovery](https://github.com/libp2p/specs/blob/master/4-architecture.md#44-discovery)
```

Comment:
this links is not quite correct. By ambient peer discovery we mean that we rely on ambient connection events, no matter how they come. Also we prescribe the existence of a driver later on.

Reply:
Right, editing in this way helps me to get a better understanding. Will make a commit with revisions.

Reply:
so can we remove this link?